recipe/meta.yaml

```diff
@@ -55,3 +88,6 @@ about:
 extra:
   recipe-maintainers:
     - jdb78
+  skip-lints:
+    - missing_wheel
+    - missing_pip_check
```

Comment:
why?

Reply:
I left comments about tensorboard above. If we skip platfroms by a selector then anaconda-linter thinks that pip check is missing for all platforms. I think it's a anaconda-linter bug

Reply:
I see, but it is not because the tests are not run that `pip` and `wheel` dependencies can be skipped, as they are not strictly test-related. even if you want to skip the checks, I see no reason to silent the wheel lint.

Reply:
Ah, I see. I'll remove it